libs/community/langchain_community/vectorstores/usearch.py

```diff
@@ -170,7 +164,7 @@ def from_texts(
             documents.append(Document(page_content=text, metadata=metadata))
 
         docstore = InMemoryDocstore(dict(zip(ids, documents)))
-        usearch = dependable_usearch_import()
+        usearch = guard_import("usearch.index")
```

Comment:
does this mean we dont need the `dependable_usearch_import` function at all?

same for the above functions

Reply:
@hwchase17 I'd prefer to completely replace them. But it is a breaking change, right? So, the whole deprecation ceremony should be in place. If you agree on this I can add deprecation decorators to all custom_import function (whenever it is applicable). But I need instructions about the decorator values.